actioncable/lib/action_cable/connection/message_buffer.rb

```diff
@@ -30,7 +30,7 @@ def process!
 
       protected
         attr_reader :connection
-        attr_accessor :buffered_messages
+        attr_reader :buffered_messages
```

Comment:
Funny, peeked in some files and the first two I clicked which used attrs, channels/base and subscriptions, did so on one line. But it doesn't matter 😁